src/psyclone/tests/psyir/tools/dependency_tools_test.py

```diff
@@ -60,11 +57,14 @@ def test_messages():
 
     dep_tools = DependencyTools()
     assert dep_tools.get_all_messages() == []
-    dep_tools._add_message("info-test", DTCode.INFO_NOT_NESTED_LOOP,
+
+    # There aren't currently any INFO messages so we invent one by simply
+    # adding one to the minimum INFO error code.
+    dep_tools._add_message("info-test", DTCode.INFO_MIN+1,
```

Comment:
Good work around :) When I noticed that the warning section became empty, I was wondering if we should remove these constants. But I am happy to leave them in and see if they might be useful as we extend the dep. tools